src/core/modules/impl/handler/JsHandlerApi.ts

```diff
@@ -16,36 +29,27 @@ export class JsHandlerApi<State> extends AbstractContractHandler<State> {
     super(swGlobal, contractDefinition);
   }
 
-  async handle<Input, Result>(
+  private async runContractFunction<Input>(
```

Comment:
can you please move private functions to the bottom of the class (i.e. first public, than private)

Reply:
of course